src/utils/extractOptions.js

```diff
@@ -0,0 +1,12 @@
+const extractOptions = data => {
+  var options = [];
```

Comment:
avoid `var` or `const` by any means

Reply:
sorry :)

it should be 
avoid `var` or `let` by any means

so always use `const`

Reply:
bc288c347ed2e128924dbbfaf66f355d8cf9daac